GOVERNANCE.md

```diff
@@ -0,0 +1,29 @@
+# Auth Working Group Governance
+
+This group is subject to the governance documented in the [TOB proposal](https://github.com/opencontainers/tob/blob/main/proposals/wg-auth.md#governance).
+
+## Code of Conduct
+
+Working group members and maintainers shall follow the [OCI code of conduct](https://github.com/opencontainers/.github/blob/master/CODE_OF_CONDUCT.md)
+
+## Discussion
+
+- Discussions take place in the [GitHub repo](https://github.com/opencontainers/wg-auth) and during [weekly meetings](https://github.com/opencontainers/wg-auth/blob/main/README.md#meetings).
+- Notes taken during the community meetings are recorded in [this hackmd document](https://hackmd.io/bGIxKAxPROi8KlwZMQioXQ), and later migrated to the [minutes](./minutes) folder via pull request.
+
+## Maintainer Operations
+
+The list of current maintainers shall be recorded in the [CODEOWNERS](https://github.com/opencontainers/wg-auth/blob/main/CODEOWNERS) document.
+
+- Maintainers shall be responsible for overseeing documentation of community decisions, consensus, and resolution of conflicts.
+- Maintainers shall provide timely feedback to requests for review and promptly merge approved Pull Requests.
+- Maintainers shall operate in good faith and trust in the community and in each other.
+- Maintainers shall keep the documentation, issues, and discussion topics as up-to-date as possible.
+- In the event a Maintainer cannot perform their duties or have been inactive for a period of time, an interim or permanent replacement shall be nominated from the community.
+- Consensus on unblocking inactivity or deadlock must be reached in a community meeting and recorded in the meeting minutes.
```

Comment:
Just curious... is there any guidance how to unblock? For example, super majority rule or overwhelming vote from the community (whatever that means). Should we add something to the governance that clarifies that?

Reply:
I copied that from the reference types working group. I'd be comfortable with a majority of maintainers resolving any conflict. I'm not too concerned since our output still needs to be approved by other existing OCI groups (TOB or an individual project like distribution-spec, and then another vote for the change to be released).